.vagrant/rgloader/loader.rb

```diff
@@ -0,0 +1,9 @@
+# This file loads the proper rgloader/loader.rb file that comes packaged
```

Comment:
This file should be ignored. It is  in the vagrant cache folder, and appears in the .gitignore file.